client/src/app/components/screens/receive/ReceiveConsentScreen.tsx

```diff
@@ -38,7 +38,14 @@ export function ReceiveConsentScreenContent(props: ContentProps) {
         </Button>
         <Text color="dark-grey" weight={400} size={14.4}>
           By using Winden you agree to the{" "}
-          <Anchor component={Link} to="/terms" color="tertiary" weight={600}>
+          <Anchor
+            component={Link}
+            to="/terms"
+            color="tertiary"
+            weight={600}
+            target="_blank"
```

Comment:
This doesn't work. Still popups warning for leaving website. If I click, still opens in the same page.